packages/blocks/src/root-block/edgeless/components/toolbar/edgeless-toolbar.ts

```diff
@@ -348,6 +349,16 @@ export class EdgelessToolbar extends WithDisposable(LitElement) {
     }
   }
 
+  // The toolbar should be disabled while edgeless AI is in progress.
```

Comment:
I think this part should be moved to the edgeless copilot widget.  Get the reference through `host.view.getWidget` and change its style or something.